notebooks/en/code_search.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>if you want you can install these libraries at the top of the notebook and only do the imports where necessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>